miniserver/routes/miniID.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express')
+const { StatusCodes } = require('http-status-codes')
+// const { NFT } = require('../../server/src/data/nft')
+const router = express.Router()
+// const { hasUserSignedBody } = require('../../server/routes/middleware')
+// const sharedUtils = require('../../shared/utils')
+const { validateID } = require('./middleware')
+const blockchain = require('../blockchain')
+
+// allows an existing user to lookup another user's address by their phone number, iff the phone number exists and the target user does not choose to hide its phone-address mapping (under `hide` parameter in settings)
+router.post('/lookup', validateID, async (req, res) => {
```

Comment:
We should not allow this arbitrarily. See the implementation in server, which verifies the request is from a registered user, using timestamped signature

Reply:
Have removed this for now we can use the servers implementation.